ec2-spot-emr/emr-spot/README.md

```diff
@@ -0,0 +1,29 @@
+# Running EMR using AWS Java v2 SDK
+## Setting up default roles
+```sh
+aws emr create-default-roles
+```
```

Comment:
May worth adding extra context of what are we trying to solve with this and adding a few links where relevant to the API used. It also make sense to give some guidance of any pre-requisite they need to run this .

I think Scott's description sets the bar of what would be ideal. https://github.com/awslabs/ec2-spot-labs/tree/master/ec2-spot-interruption-logging-insights. Some extra description and context definitely would make a difference